DuplicateAssemblyScanner/DuplicateAssemblyScanner/Structs/ModAssembly.cs

```diff
@@ -0,0 +1,50 @@
+namespace DuplicateAssemblyScanner.Structs {
+    using DuplicateAssemblyScanner.Util;
+    using System.Reflection;
+    using static ColossalFramework.Plugins.PluginManager;
```

Comment:
Prefer not to use `using static`. This might confuse the readers. Only use `using static` for `static` classes containing `public const` fields, or better do not use `using static` at all.

Reply:
Doing so breaks the `<see cref="PluginInfo.modPath"/>` references in comments.

Reply:
Removed the `see` comments.

Reply:
You just can use `<see cref="ColossalFramework.Plugins.PluginManager.PluginInfo.modPath"/>